tests/test_parser.py

```diff
@@ -641,15 +641,15 @@ def test_expire_by_major_version():
             "category": {
                 "metric_expired_version": {
                     "type": "boolean",
-                    "expires": 18,
+                    "expires": 7,
                 },
                 "metric_expired": {
                     "type": "boolean",
                     "expires": "expired",
                 },
                 "metric": {
                     "type": "boolean",
-                    "expires": 7,
+                    "expires": 18,
```

Comment:
Can you add one more metric that expires in v15?
Just so we have a test that's explicitly the edge case ("expires" defines the first version in which this metric is expired)

Reply:
Sure, done!